pkg/sdk/sdk.go

```diff
@@ -279,9 +279,9 @@ type SDK interface {
 	// ResetPasswordRequest sends a password request email to a user.
 	//
 	// example:
-	//  err := sdk.ResetPasswordRequest("example@email.com")
+	//  err := sdk.ResetPasswordRequest(context.Background(), "example@email.com", "https://example.com")
 	//  fmt.Println(err)
-	ResetPasswordRequest(ctx context.Context, email string) errors.SDKError
+	ResetPasswordRequest(ctx context.Context, email, host string) errors.SDKError
```

Comment:
This should not be part of the SDK, but rather a configuration of the users service. I.e. you do not determine he host (URL, really) via (SDK) request.

Reply:
@dborovcanin should it be removed as part of the request? https://github.com/absmach/supermq/blob/e842bfaab6c359423d1544ee428b8a1a9bee4944/users/api/requests.go#L285

Reply:
Yes.